Doc/library/threading.rst

```diff
@@ -82,7 +82,7 @@ This module defines the following functions:
    Its value may be used to uniquely identify this particular thread system-wide
    (until the thread terminates, after which the value may be recycled by the OS).
 
-   .. availability:: Windows, FreeBSD, Linux, macOS.
```

Comment:
You also have to update the attribute:
https://docs.python.org/dev/library/threading.html#threading.Thread.native_id

Hum, maybe the atttribute availability can be enhanced as ".. availability:: Required :func:`get_native_id` function." to avoid redundance.